api/envoy/api/v2/cluster/outlier_detection.proto

```diff
@@ -80,4 +80,30 @@ message OutlierDetection {
   // used to disable ejection or to ramp it up slowly. Defaults to 0.
   google.protobuf.UInt32Value enforcing_consecutive_gateway_failure = 11
       [(validate.rules).uint32.lte = 100];
+
+  // Determines whether to distinguish local origin failures from external errors. If set to true
+  // the following configuration parameters are taken into account:
+  // consecutive_local_origin_failure, consecutive_local_origin_failure
+  // enforcing_consecutive_local_origin_failure and enforcing_local_origin_success_rate.
+  // Defaults to false.
+  google.protobuf.BoolValue split_external_local_origin_errors = 12;
```

Comment:
This can be a standard `bool` value.

Reply:
I tried to change it to _bool_, but compiler complains that
PROTOBUF_GET_WRAPPED_OR_DEFAULT macro cannot resolve _has_##field_name()_ which maps to non-existing _has_split_external_local_origin_errors_.
I can investigate it further if it is important not to use _google.protobuf.BoolValue_.

Reply:
You shouldn't need any default for a `bool` as the default is false. That's why we should use it here.

Reply:
OK. got it. Thanks!